doc/jenkins-conda.rst

```diff
@@ -0,0 +1,168 @@
+=================
+Conda Jenkinsfile
+=================
+
+This example is pretty much ready to go.
```

Comment:
> This example is pretty much ready to go.

Doesn't sound very professional. 

Please remove this phrase and replace it with an explanation of what "Conda Jenkisfile" is intended for. I would expect to have some explanation of what we use conda for, what we use Jenkins for and what we user Jenkins + Conda for. Then it should also say that this page contains templates for developers to integrate their software with our system and how this is accomplished (copy the file bellow and make the appropriate edits)